src/main.rs

```diff
@@ -120,31 +122,31 @@ fn main() {
     {
         Ok(a) => a,
         Err(error) => {
-            return fatal_error(format!("Error parsing arguments: {}", error));
+            panic!("error parsing arguments: {}", error);
```

Comment:
I don't know if we want to `panic!` instead of using this homemade function. With the latest "logging" changes I just merged, the `fatal_error` does structured logs container the error message. This would no longer happen with `panic!`.

Maybe we could replace `fatal_error` with a macro that does something like:

```rust
error!(...)
panic!(...)
```

Reply:
Yes, going back to the home made function for now.